src/pages/zmest/handleSendEmailClick.js

```diff
@@ -0,0 +1,86 @@
+let timeoutTimer
+let emailWasSentRecently
+
+function getUtmData() {
+  const url = window.location.href
+  const utmList = url.split('utm').slice(1)
+  let result
+
+  if (utmList.length > 0) {
+    result = utmList.map(item => {
+      return item
+        .replace('_source=', 'Campaign Source: ')
+        .replace('_medium=', 'Campaign Medium: ')
+        .replace('_campaign=', 'Campaign Name: ')
+        .replace('_term=', 'Campaign Term: ')
+        .replace('_content=', 'Campaign Content: ')
+        .replace('&', '')
+    })
+    result = result.join('; ')
+  } else {
+    result = `Not Identified (doesn't exist)`
+  }
+
+  return result
+}
+
+const changeButtonStatus = status => {
+  const defaultButton = document
```

Comment:
Document doesnt exist in prerender process. Try npm build and where will be an error